src/ports/system/model.ts

```diff
@@ -0,0 +1,42 @@
+import type { SessionConfig } from 'h3';
```

Comment:
This File only adds typings. Maybe model and interface should be merged. As we don't follow a mvc-like pattern